### ISSUE TITLE
Update Android section of Processing Wiki

### ISSUE THREAD

**Issue by processing-bugs** [processing/processing-android#34]
<a href="https://github.com/jeffThompson"><img src="https://avatars.githubusercontent.com/u/509469?" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [jeffThompson](https://github.com/jeffThompson)**
_Tuesday May 21, 2013 at 14:57 GMT_
_Originally opened as https://github.com/processing/processing-android-archive/issues/36_

---

Since the Android section of the Processing Wiki seems to be the main source of install, setup, and coding instructions for the Android mode, I'd like to suggest that it could use a fairly extensive revision. I'd be happy to volunteer for the job, but wanted to first engage the other stakeholders in conversation and see what made the most sense, or if there were opinions about what to add/change.

My suggestions are:
- The single page is now much too long, and should be broken into separate pages
- I would suggest a page for install/setup/testing, basics, and maybe separate pages for individual examples like touch, internet, sensors, icons, etc like we have for the desktop version
- If anyone has experience uploading a Processing-built app to Google Play, I think that would be a very interesting tutorial too
- An overall cleanup of the text to be easier to read (I like verbose, but it can be a bit hard to follow, speaking from my experience setting up the IDE from the Wiki)

I would also guess that some of the new additions might be helpful filling out the example sketches - I'd be delighted to help with that as well.

**Comment by codeanticode** [comment]
Wiki has been updated, and more materials will be added soon
